gregoriotex/index.html

```diff
@@ -54,26 +54,26 @@
     context = 'gregoriotex';
     title = 'GregorioTeX';
 </script>
-<h1>GregorioTeX</h1>
+<h1>Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></h1>
 
-<p>This page describes quickly the origin and the construction of GregorioTeX. You can also take a look at the <a href="introduction.html">GregorioTeX introduction page</a> or at the <a href="details.html">GregorioTeX detailed description page</a>. If it is the first time you are dealing with TeX, I advise you to take a look at the <a href="tex.html">TeX page</a>.</p>
```

Comment:
You know, I'm not actually sure where this link should point to.  Perhaps we're missing a page?